### ISSUE TITLE
📚 Documentation: 

### ISSUE THREAD

**Issue by akashgreninja** [appwrite/appwrite#5187]
### 💭 Description

During the creation of a document we have to create attributes as well else it'll throw an error 

`Invalid Document Structure: Unknown attribute <attribute>`

Though creating attributes is required it is not put in the  `create  document`   documentation  as required 

Creating a collection is given as required and redirects us to the same 
but there is nothing about making an attribute compulsory it will be  hard for the new users to figure that out as there are no answers on StackOverflow regarding the same 

Please make the changes as it will improve the user experience 

![Screenshot 2023-03-05 113351](https://user-images.githubusercontent.com/84433561/222944420-c715eee8-bce5-4430-b507-e93d6edc1f0a.png)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@akashgreninja thanks for opening this issue! 🙏🏼 Did you happen to read the docs at the top of that [page](https://appwrite.io/docs/client/databases)? Or the [Databases Guides](https://appwrite.io/docs/databases)?

**Comment by akashgreninja** [comment]
> @akashgreninja thanks for opening this issue! 🙏🏼 Did you happen to read the docs at the top of that [page](https://appwrite.io/docs/client/databases)? Or the [Databases Guides](https://appwrite.io/docs/databases)?

Database Guidelines actually under create documents

**Comment by stnguyen90** [comment]
> Database Guidelines actually under create documents

@akashgreninja, sorry, what do you mean? Are you saying you didn't read the other docs I mentioned and you only looked at the description of Create Documents?

**Comment by akashgreninja** [comment]
> > Database Guidelines actually under create documents
> 
> @akashgreninja, sorry, what do you mean? Are you saying you didn't read the other docs I mentioned and you only looked at the description of Create Documents?

Yes I did read the description of create Documents and there was nothing about attributes being necessary, It would be better if its written under the create documents section as well the image below is the only reference to create attributes are required for creating documents
![Screenshot 2023-03-08 095725](https://user-images.githubusercontent.com/84433561/223619675-64791a0d-dd04-4e0e-9cbf-860862bfbb66.png)

**Comment by gewenyu99** [comment]
@stnguyen90 I'm gonna fix this in docs but also improve that error message
